test/suites/smoke-test-common/test-consistency-services-payment.ts

```diff
@@ -0,0 +1,67 @@
+import "@tanssi/api-augment";
+import { beforeAll, describeSuite, expect } from "@moonwall/cli";
+import { ApiPromise } from "@polkadot/api";
+import { paraIdTank } from "util/payment";
+
+describeSuite({
+    id: "S09",
+    title: "Check services payment consistency",
+    foundationMethods: "read_only",
+    testCases: ({ it, context }) => {
+        let api: ApiPromise;
+        let runtimeVersion;
+        const costPerSession = 100_000_000n;
+        const costPerBlock = 1_000_000n;
+        let blocksPerSession;
+
+        beforeAll(async () => {
+            api = context.polkadotJs();
+            runtimeVersion = api.runtimeVersion.specVersion.toNumber();
+            const chain = api.consts.system.version.specName.toString();
+            blocksPerSession = chain == "Dancebox" ? 300n : 5n;
+        });
+
+        it({
+            id: "C01",
+            title: "All scheduled parachains should be able to pay for at least 2 sessions",
+            test: async function () {
+                if (runtimeVersion < 500) {
+                    return;
+                }
+                const existentialDeposit = await api.consts.balances.existentialDeposit.toBigInt();
+
+                const pending = await api.query.registrar.pendingParaIds();
```

Comment:
Registered para ids are allowed to run out of credits so this test will fail, but we can look at which para ids have been assigned collators using `collatorAssignment.pendingCollatorContainerChain`. This is because the call to `remove_para_ids_with_no_credits` is done in the `collatorAssignment` pallet.

Reply:
Done in https://github.com/moondance-labs/tanssi/pull/437/commits/9b32ab1875025c0372e2057ead86a227c351b8f7